hpOneView/resources/networking/network_sets.py

```diff
@@ -86,13 +86,13 @@ def delete(self, resource, force=False, blocking=True):
             force:
                  If set to true the operation completes despite any problems with
                  network connectivity or errors on the resource itself. The default is false.
-            blocking:
-                Wait task completion
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout do not abort the operation
+                in OneView, just stop waiting its completion.
```

Comment:
I think a better description for the timeout text "The timeout do not abort the operation in OneView, just stop waiting its completion." is "The timeout does not abort the operation in OneView, just stops waiting for its completion.